webhook-integrations/Facebook/notes.md

```diff
@@ -0,0 +1,36 @@
+# Facebook Integration Investigation
+
+[Facebook](https://www.developers.facebook.com) is a social networking service that also allows developers
+to track analytics of their apps.
+
+Facebook has a very detailed notification model for sending notifications to users.
+
+Facebook has no integrations with Slack, but a webhook URL can be configured to receive payloads.
+
+
+* The Notification Model of Facebook is-
+  <br>
+  ![Notification Model](webhook_topics.PNG)
+
+For each of the above parameters, there are at least 15 cases (some have about 40) for which 
+Facebook can send notifications to the Webhook URL with a verification token.
+
+* Some Cases of The 'Application' parameter are-
+  <br> 
+  ![Application Cases](app_cases.PNG)
+ 
+When one's App/Website is configured to use the Facebook SDK, (Configuration Help can be found [here](https://developers.facebook.com/products/analytics/quickstarts))
+Facebook then logs the events that happen in the app/website & sends out notifications for
+the parameters that have been configured to use the webhook. Also, different cases are invoked in the 
+parameter only if one subscribes to those cases.
+e.g.
+If I configure my App to use the Facebook SDK, & configure the parameter 'User' for sending out 
+notifications & I subscribe to the 'email' case in 'User', then Facebook sends out a notification
+to the webhook every time a new user email is added/deleted from the App.
+
+## Links
+Facebook For Developers: https://developers.facebook.com
+
+## Other Notes
+The 4 parameters were chosen considering their use if integrated with Zulip, & the cases for each
```

Comment:
What exactly are these 4 parameters? If you mean "4 different webhook notification types", you should specify them individually.